### PR TITLE
Avatars/Community Avatar Component

### LINKED ISSUE #20130: Avatars/Community-Avatar
Design: https://www.figma.com/design/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=8824-149727&m=dev

### PR DESCRIPTION
**Pull request by ajayesivan**
fixes #20130 

### Summary

Implements the [Avatars/Community Avatar](https://www.figma.com/design/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=8824-149727&m=dev) component

### Testing notes
Manual QA can be skipped since this PR only adds a quo component and doesn't introduce any changes outside Quo Preview.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
None

### Steps to test
Quo Preview -> Avatars -> Community Avatar

status: ready

### CODE REVIEW

File: src/quo/components/avatars/community_avatar/schema.cljs [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:size {:optional true} [:maybe [:enum :size-32 :size-24]]]
+      [:image :schema.common/image-source]
+      [:container-style {:optional true} [:maybe :map]]]]]
+   :any])
```
Comment (clauxx) [inline src/quo/components/avatars/community_avatar/schema.cljs]: The schema can be in the same ns with the component if it's small like this one. We tend to move it to a separate ns if you end up composing the schema from other smaller schema definitions, which takes some space and pollutes the component ns.

**Review by clauxx — APPROVED** [review]
Looks good 👍

File: src/quo/components/avatars/community_avatar/view.cljs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+(ns quo.components.avatars.community-avatar.view
+  (:require [quo.components.avatars.community-avatar.schema :as community-avatar.schema]
+            [quo.components.avatars.community-avatar.style :as style]
+            [react-native.core :as rn]
+            [schema.core :as schema]))
+
+(defn view-internal
```
Comment (OmarBasem) [inline src/quo/components/avatars/community_avatar/view.cljs]: `defn- view-internal`

**Review by OmarBasem — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

File: src/quo/components/avatars/community_avatar/schema.cljs (hunk 1 above)
Comment (ajayesivan) [reply src/quo/components/avatars/community_avatar/schema.cljs]: Personally, I prefer keeping schemas in a separate file. In my workflow, I find it very easy to locate and work with component schemas when they are defined separately. However, I don't have strong objections to writing small schemas within the component view file itself. It seems we still haven't reached a consensus on this.

**Comment by ajayesivan** [comment]
Hi @Francesca-G, could you please do a design review for this PR?

Thanks!

**Comment by yevh-berdnyk** [comment]
> ## 83% of end-end tests have passed

Hi @ajayesivan, thanks for the PR! E2E failures are not related

**Review by Francesca-G — APPROVED** [review]
Nice job ✨

**Pull request merged**